### PR TITLE
Clarify pull request and approval process

### PR DESCRIPTION
**Pull request by bion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,15 @@ Development tasks are managed in the github issues for this repository. When you
...
+It's easy for the intention of code review comments to be unclear or get misinterpreted. To help with communication, reviewers are encouraged to use [conventional comments](https://conventionalcomments.org/) and explicitly indicate that comments are `(blocking)`, where the discussion must be resolved for PR to be merged, or `(non-blocking)` where resolving the discussion is optional for the implementer.
+
+#### Approval and merging
+
+Reviewers should grant approval if they do not feel additional review is necessary before merging. This does not necessarily mean no more changes are required before merging, but that any further changes are expected to be minor enough to not require review.
+
+If the pull request does not require additional changes, the reviewer should merge it immediately after approving. Otherwise, once they have addressed all comments marked `(blocking)` or `nit`, the pull request author should either merge if able or re-request review and merging from a maintainer if not. Authors are encouraged to at least reply to `(non-blocking)` and `(if-minor)` comments if they do not address them with code changes.
```
Comment (carolinedanzi) [inline README.md:37]: question: will "the reviewer should merge it" be our standard moving forward? I feel the author has been the one merging so far

**Review by carolinedanzi — APPROVED** [review]
Thank you for clarifying, this looks nice!

File: README.md (hunk 1 above)
Comment (bion) [reply README.md:37]: I think it should be that the reviewer merges it unless there are minor things for the author to change post-approval. In open source it's generally more common for maintainers to do all merging.

Comment (carolinedanzi) [reply README.md:37]: Thanks for clarifying! Sounds good to me

**Pull request merged**